setup.py

```diff
@@ -82,6 +82,7 @@ def find_version(*file_paths):
             "nbclient<=0.7.3",
             "nbformat<=5.8.0",
             "nbsphinx<=0.9.1",
+            "lxml_html_clean",
```

Comment:
Orthogonal dependency fix, see e.g. https://github.com/assafelovic/gpt-researcher/pull/424/files for a similar fix.

Reply:
Hmm that link doesn't really have much information on what exactly this fixes / why this dependency is needed. Is there more info on this?

Reply:
There was a change in some package required for the sphinx compilation recently, which outsourced part of the functionality into this new package (there's more info in the description tab of the linked PR). 

The doc compilation will fail for any upcoming commit without this, so this is independent of the other changes made here.

Reply:
Copying over the error message from the link, which was raised on this commit as well before I added the line:

ImportError: lxml.html.clean module is now a separate project lxml_html_clean.
Install lxml[html_clean] or lxml_html_clean directly.